cmd/defaults.yaml

```diff
@@ -834,6 +850,28 @@ AuditLogRetention: 0s
 
 InternalAuthZ:
   RolePermissionMappings:
+    - Role: "SYSTEM_OWNER"
+      Permissions:
+        - "system.instance.read"
+        - "system.instance.write"
+        - "system.instance.delete"
+        - "system.domain.read"
+        - "system.domain.write"
+        - "system.domain.delete"
+        - "system.debug.read"
+        - "system.debug.write"
+        - "system.debug.delete"
+        - "system.feature.write"
+        - "system.limits.write"
+        - "system.limits.delete"
+        - "system.quota.write"
+        - "system.quota.delete"
+        - "system.iam.member.read"
```

Comment:
Should system.iam.member.read also be part of SYSTEM_OWNER_VIEWER?

Reply:
added to SYSTEM_OWNER_VIEWER